arelle/XmlValidateParticles.py

```diff
@@ -9,9 +9,13 @@
                                    ModelAll, ModelChoice, ModelSequence, 
                                    ModelAny, anonymousTypeSuffix)
 from arelle.ModelObject import ModelObject, ModelAttribute
+from arelle.XbrlConst import xsd
 from arelle.XmlValidate import validate
 
+
 def validateElementSequence(modelXbrl, compositor, children, ixFacts, iNextChild=0):
+    if compositor.modelDocument.targetNamespace == xsd:
+        return None
```

Comment:
We can't return None here because of [this](https://github.com/Arelle/Arelle/blob/ee333d266da4b793e94e4b1e47219f23d38647c5/arelle/XmlValidateParticles.py#L52)

Reply:
I have tested the new fix and it works as expected